documentation/api-scenario/references/ErrorCodeReference.md

```diff
@@ -119,3 +119,36 @@ Example: The sku in request parameters is `default`, but actual return is `stand
 }
 
 ```
+
+## INCORRECT_LRO_FINAL_STATE_VIA
+
+This error is reported when the long running operation defined with 'final-state-via' in swagger but the initial request doesn't return a corresponding header described in the [x-ms-long-running-operation-options](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-long-running-operation).
+
+**How to fix**
+1. if a long running operation flagged with 'final-state-via': 'location or azure-async-operation or operation-location', the response of the inital request MUST include a corresponding header which contains the url for getting the final result of the operation.
+
+1. if a long running operation flagged with 'final-state-via': 'original-uri', then the operation should be a PUT/PATCH/DELETE operation of an ARM resource type and a GET operation should also be present under the corresponding swagger 'path'.
+
+
+## INVALID_FINAL_GET_RESULT
+
+1. This error is reported when the final-get result of the long running operatoin doesn't match the swagger schema. you can see [x-ms-long-running-operation-options](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-long-running-operation) for define final result.
+
+**How to fix**
+1. Check if the 'final-state-via' of the LRO is correct, you may refer to the [INCORRECT_LRO_FINAL_STATE_VIA](./ErrorCodeReference.md#incorrect_lro_final_state_via) for details.
+2. If the 'final-state-via' of the LRO is correct, you should update the LRO response to fix the response code and/or schema.
+ Generally,if the final-get response body is empty, a '204' response code with empty schema should be defined for the LRO in the swagger , like
+``` json
+"204": {
+   "description": "NoContent. Successfully requested delete operation, but the response is intentionally empty."
+}
+```
+   Or a '200' response code with correct schema should be defined for the LRO in the swagger, like
```

Comment:
in this case, the error code will be normal output from schema validation, like `INVALID_TYPE`, etc. So it seems no need to define this errorCode?